kernal/cbm/channel/x16additions.s

```diff
@@ -24,7 +24,7 @@ extapi_getlfs:
 ;   fnadr+fnlen: filename pointer (via setnam)
 ;   fa: device (via setlfs)
 ; outputs
-;   returns m=1, x=1
+;   returns m=0, x=0
```

Comment:
I haven't programmed for the 816. If I understand correctly, m and x are set at the beginning of this function (SEP #$30).

The flags are cleared at lines 41 and 57 before RTS.

If R1L is not 0, execution continues from label @2. I can't see that that line of code always goes by a REP instruction. In some cases the function is ended with JMP break. 

I think "break" is the function in kernal/cbm/channel/save.s. As far as I can see, it doesn't call REP.

So, is m and x always set to 0 when returning from this function?